deploy.sh

```diff
@@ -8,15 +8,23 @@ LOG_FILE=deployment.log
 
 # Defaults for production.
 # Example for development:
-#   PORT=8100 OPENFISCA_PORT=12100 ./deploy.sh aah
+#   PORT=8100 OPENFISCA_PORT=12100 ./deploy.sh
 # Example for production:
 #   PORT=8000 OPENFISCA_PORT=2000 PUBLIC_HOST=mes-aides.gouv.fr PROTOCOL=https ./deploy.sh
-TARGET_BRANCH=${1:-master}
-PORT=${PORT:-8000}
-OPENFISCA_PORT=${OPENFISCA_PORT:-12000}
-PUBLIC_HOST=${PUBLIC_HOST:-$TARGET_BRANCH.mes-aides.sgmap.fr}
-PROTOCOL=${PROTOCOL:-http}
+USER=`whoami`
+TARGET_BRANCH=${USER#mes-aides-}
+source current_config.env || echo "No current_config.env file found."
+PORT=${PORT:-$CURRENT_PORT}
+OPENFISCA_PORT=${OPENFISCA_PORT:-$CURRENT_OPENFISCA_PORT}
 
+if ! [[ -n $PORT && -n $OPENFISCA_PORT ]]
```

Comment:
Shouldn't this be `||` rather than `&&`? If any one of them isn't defined, we have a problem, right?